src/arrays.js

```diff
@@ -1,61 +1,64 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) {
+    return array[index - array.length]
+  };
+  return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join(',');
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(word => word.toUpperCase())
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings.map(strings => strings.split('').reverse().join(''));
 };
 
 const onlyEven = numbers => {
-  // your code here
+  return numbers.filter(numbers => numbers % 2 == 0);
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  return array.filter((animal, indexArray) => indexArray !== index);
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+  return strings.filter(strings => /^[aeiou]/i.test(strings));
```

Comment:
`strings.filter(strings...` should be change to `strings.filter(string...` or something similar, as strings is already called in the upper scope.